### DISCUSSION TITLE
how do you add a proxy?

### DISCUSSION THREAD

**Discussion by Ender-Ghost-Craft** [red-stone-network/clockwork#67] [Q&A]
what url would I add to the proxy settings, its not very clear? the code is
self.__uv$config = {
prefix: '/school/',
bare: '/bare/',
encodeUrl: Ultraviolet.codec.xor.encode,
decodeUrl: Ultraviolet.codec.xor.decode,
handler: '/uv/uv.handler.js',
bundle: '/uv/uv.bundle.js',
config: '/uv/uv.config.js',
sw: '/uv/uv.sw.js',
da: 'a'
};

and the proxy is 69.eightgrade.com/

**Comment by ToasterPanic** [comment]
``https://69.eightgrade.com/school/``

**Reply by Ender-Ghost-Craft** [reply]
I cant use  the store, even with a proxy, can you please change the link, or add a way to add apps from settings with a code?
